static/js/twitter-filter.js

```diff
@@ -0,0 +1,30 @@
+function twitter_filter() {
+  var twitter_handles = []
+  for (var i = 0; i < document.getElementsByClassName('org-info').length; i++) {
+    var org_info = document.getElementsByClassName('org-info')[i]
+    for (var j = 0; j < org_info.children.length; j++) {
+      if (org_info.children[j].href !== undefined) {

```

Comment:
More safe with `typeof` instead of compare it directly.

Reply:
Or just `if (org_info.children[j].href) {`